### DISCUSSION TITLE
There is any sea_orm::ColumnType for varchar?

### DISCUSSION THREAD

**Discussion by fabioluciano** [SeaQL/sea-orm#771] [Q&A] [answered]
I was searching for some way to define a column with varchar(32) and i didn`t find.

**Comment by nickb937** [comment, marked as answer]
In your `ColumnTrait` definition, you apply a numeric value to the `ColumnType:String` enum to limit it, which will become `VARCHAR(32)` in the DDL:

```rust
Self::ColName => ColumnType::String(Some(32)).def()
```

**Reply by fabioluciano** [reply]
Thankyou @nickb937
